sentry-android-core/src/main/java/io/sentry/android/core/InternalSentrySdk.java

```diff
@@ -44,9 +45,9 @@ public final class InternalSentrySdk {
   @Nullable
   public static IScope getCurrentScope() {
     final @NotNull AtomicReference<IScope> scopeRef = new AtomicReference<>();
-    // TODO [HSM] should this retrieve combined scope?
```

Comment:
Yes, at least for cross platform SDKs until they support different scope types separately themselves.